src/device/usbd.c

```diff
@@ -37,6 +37,10 @@
 #define CFG_TUD_TASK_QUEUE_SZ   16
 #endif
 
+#ifndef CFG_TUD_EP_MAX
+#define CFG_TUD_EP_MAX          8
```

Comment:
let's default it to `9` for now so that nrf5x port doesn't have to manually defined it. Later on we could have an better #if according to the port so that we don't waste SRAM on some MCU (e.g some stm32 only support 4/5 pairs of endpoints). However the memory consumed is too small ( few bytes ) to worth the effort for now.

Reply:
ok

Reply:
no problem, my original code is not too consistent with if else between stall and clear stall. I must write it at 2 different time and it is easy to overlook.